nix/packages.nix

```diff
@@ -0,0 +1,102 @@
+{ inputs }:
+
+let
+  systems = [
+    "x86_64-linux"
+    "aarch64-linux"
+    "aarch64-darwin"
+  ];
+in
+inputs.nixpkgs.lib.genAttrs systems
+  (
+    system:
+    let
+      pkgs = inputs.nixpkgs.legacyPackages.${system};
+      lib = pkgs.lib;
+      defaultPackages = import ./default-pkgs.nix { inherit pkgs; };
+
+      # prefetch-npm-deps react-display/package-lock.json
+      npmDeps = pkgs.fetchNpmDeps {
+        src = "../react-display/.";
+        hash = "sha256-QdTanJu5rjozXrqAkiMb+RisMaqKkjwYtv9+IAkosUU=";
+      };
+
+      targets = [
+        "linux/amd64"
+        "linux/arm64"
+        "darwin/arm64"
+      ];
+
+    in
+    {
+      go-signs = pkgs.buildGoModule rec {
+        pname = "go-signs";
+        version = "unstable";
+        src = builtins.path { path = ../.; };
+        goPackagePath = "github.com/kylerisse/go-signs";
+
+        vendorHash = "sha256-mrHfOS9EoM8o9RXQBYVLGFR+uGITI3v/aRAKBq3/wc0=";
+
+        nativeBuildInputs = defaultPackages ++ [ npmDeps ];
```

Comment:
You shouldnt need to duplicate `defaultPackages` and should be able to just go into `buildInputs`

nativeBuildInputs: build-time deps
buildInputs: runtime deps

Reply:
ah, thanks for the call out